kahuna/public/js/crop/view.html

```diff
@@ -52,30 +52,9 @@
             </form>
         </div>
 
-        <label class="top-bar-item clickable side-padded" gr-tooltip="landscape [l]">
-            <input type="radio" ng:model="ctrl.aspect" value="{{ctrl.landscapeRatio}}" ng-checked="ctrl.landscapeChecked || ctrl.cropType === 'landscape'" ng-disabled="ctrl.cropType" name="aspect" />
-            <gr-icon-label gr-icon="crop_landscape">Landscape</gr-icon-label> ({{::ctrl.getRatioString(ctrl.landscapeRatio)}})
-        </label>
-
-        <label class="top-bar-item clickable side-padded" gr-tooltip="portrait [p]">
-            <input type="radio" ng:model="ctrl.aspect" value="{{ctrl.portraitRatio}}" ng-disabled="ctrl.cropType" name="portrait ratio" />
-            <gr-icon-label gr-icon="crop_portrait">Portrait</gr-icon-label> ({{::ctrl.getRatioString(ctrl.portraitRatio)}})
-        </label>
-
-        <label class="top-bar-item clickable side-padded" gr-tooltip="video [v]">
-            <input type="radio" ng:model="ctrl.aspect" value="{{ctrl.videoRatio}}" ng-checked="ctrl.cropType === 'video'" ng-disabled="ctrl.cropType" name="video ratio" />
-            <gr-icon-label gr-icon="crop_16_9">Video</gr-icon-label> ({{::ctrl.getRatioString(ctrl.videoRatio)}})
-        </label>
-
-        <label class="top-bar-item clickable side-padded" gr-tooltip="square [s]">
-            <input type="radio" ng:model="ctrl.aspect" value="{{ctrl.squareRatio}}" ng-disabled="ctrl.cropType" name="square ratio" />
-            <gr-icon-label gr-icon="crop_square">Square</gr-icon-label> ({{::ctrl.getRatioString(ctrl.squareRatio)}})
-        </label>
-
-        <label class="top-bar-item clickable side-padded" gr-tooltip="free-form [F]">
-            <input type="radio" ng:model="ctrl.aspect" value="{{ctrl.freeRatio}}" ng-disabled="ctrl.cropType" name="free ratio" />
-            <gr-icon-label gr-icon="crop_free">Free-form</gr-icon-label>
-        </label>
+        <div class="inline-block side-padded">
+          <gr-radio-list gr-for="crop" gr-options="ctrl.cropOptions" gr-selected-option="ctrl.cropType"></gr-radio-list>
```

Comment:
basically what I really wanted to do 😄 

![img](https://media.giphy.com/media/3o7TKr2xg9OWcU8DWo/giphy.gif)